package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -677,7 +679,8 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
          *
          * @name PromptPanel#_submit
          * @method
-         * @param {Object} options
+         * @param {Object}  [options]        Aditional configuration options.
+         * @param {Boolean} [options.isInit] Flag indicating if submit is being executed during initialization.
```

Comment:
Is this the only option? If so, then just make it a parameter, instead of wrapped in a JSON